src/CarwowTheme/ProductListing.elm

```diff
@@ -50,6 +50,68 @@ type alias ProductListing =
     , dealer : DealerProperties
     }
 
+
+{-| Placeholder
+-}
+type alias GroupedLeaseDealListing =
+    { makeName : String
+    , makeSlug : String
+    , monthlyRentalPounds : String
+    , monthlyRentalPennies : String
```

Comment:
is pennies a good var for multiple currencies?

Reply:
good point will just do firstPart and secondPart